game/gbtd_gbr0.ksy

```diff
@@ -0,0 +1,230 @@
+meta:
+  id: gbr0
+  endian: le
+  encoding: ascii
+  title: GBR file
+  license: MIT
+  application: GameBoy Tile Designer, GBTD
+  file-extension:
+    - gbr
+  tags:
+    - game
+
+doc: |
+  GBR0 files are used by GameBoy Tile Designer
+  to store tilemaps created with the program.
```

Comment:
It'd be nice to provide a link to a simple test file. If there is no in the  Net, it is possible to attach it to GH issue (not necessarily to have a link in the issue, you can jhst drag'n'drop a file, GH will generate a link, then this link can be just inserted into a spec).

Reply:
I have some test files as part of the repo I'm doing this development in: https://github.com/ZetaTwo/gameboy-project/tree/master/gbr-parser would that work or should it be more stand-alone?

Reply:
Completely OK.